app/assets/javascripts/components/timur.js.jsx

```diff
@@ -1,6 +1,12 @@
+import createLogger from 'redux-logger'
```

Comment:
using the envify node package with browserify-rails should default the NODE_ENV to Rails.env automagically. https://github.com/browserify-rails/browserify-rails